### DISCUSSION TITLE
Detect what's triggering the @change event

### DISCUSSION THREAD

**Discussion by lucianobosco** [advanced-cropper/vue-advanced-cropper#212] [Q&A] [answered]
I'm facing an issue, not a bug, when implementing the cropper.
When my resource is loaded I save the original object fetched from API. From that point, any change made by the user is propagated to a cloned object.
Said that, I can easily know if any object property is dirty by comparing both objects, the original one and the cloned one mutated by the user.

When cropper mounts, it dispatches the @change event twice. I assume they are because mounting component and image is loaded.
That @change event mutates my cloned object, therefore I have a dirty object as per my logic as soon the cropper mounts.

I'd like to be able to emit or recognize the @change event only when user is in fact interacting with the cropper component.
Any advice will be appreciated

**Comment by lucianobosco** [comment, marked as answer]
OMG, asked twice the same. Sorry about that
https://github.com/advanced-cropper/vue-advanced-cropper/discussions/199

**Reply by Norserium** [reply]
It's okay. Good luck!
